app/enums/Role.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\enums;
+
+enum Role: string
+{
+    case Admin = 'Admin';
+    case Member = 'Member';
+    case Moderator = 'Moderator';
+
+    public static function values(): array
+    {
+        return array_column(self::cases(), 'value');
+    }
```

Comment:
You can add functions to validate if the model who casted the enum is actually a mod, member or admin.